nautobot/circuits/navigation.py

```diff
@@ -0,0 +1,105 @@
+from nautobot.extras.nautobot_app import NavMenuButton, NavMenuItem, NavMenuTab, NavMenuGroup
+from nautobot.utilities.choices import ButtonColorChoices
+
+
+menu_tabs = (
+    NavMenuTab(
+        name="Circuits",
+        weight=500,
+        groups=(
+            NavMenuGroup(
+                name="Circuits",
+                weight=100,
+                items=(
+                    NavMenuItem(
+                        link="circuits:circuit_list",
+                        link_text="Circuits",
```

Comment:
Should `NavMenuItem` also have a `weight` that influences their ordering? Is this needed to potentially allow plugins to insert buttons into an existing `NavMenuGroup`?